bin/next_downtime_id

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+cd "$(dirname "$0")"
+datadir=$(git rev-parse --show-toplevel)/topology
```

Comment:
Minor: `datadir` could just as well be `../topology` rather than calling `git`; or appended to the `cd` line.

Reply:
The git command needs to be run from the right dir so the cd is necessary.

Reply:
No, i meant, either
```
cd "$(dirname "$0")"
datadir=../topology
```
or just
```
cd "$(dirname "$0")"/../topology
```
(and omit `$dirname` in the glob)